### ISSUE TITLE
Rename `time` field on Breakdown object to `duration`

### ISSUE THREAD

**Issue by ldeboer** [UniversalScheduleStandard/UniversalScheduleStandard#5]
I would propose renaming the `time` field on Breakdown object to `duration`. Duration is a more accurate description and using the name "time" could mislead people into thinking it is specifying the time of day (such as 12:30pm) rather than the duration.

```
{
  "id": string | UUID value | required,
  "comments": string | can be null,
  {....}
  "duration": number | millisecond duration to shoot scene | can be null,
   {....}
}
```

Luke @ SetHero ([luke@setheroapp.com](mailto:luke@setheroapp.com))

**Comment by modemmute** [comment]
Renaming `time` to `duration` makes sense and would be a simple and potentially helpful change to the standard. I can definitely see it decreasing the possibility of misusing the key when implemented.

Will leave this issue open for a while to allow others to comment.

**Comment by UniversalScheduleStandard** [comment]
This suggestion was accepted and is included in v0.9.1
